File: C Programming/problem on numbericals/problem13.c
/*
13.Write a program which accept number from user and print that number of $ & * on screen. 

	Author : Milind Nalawade
	
	Date : 25/06/2021
	
	Language : C programming
	
*/

#include<stdio.h>

//Logic

void Print(int ino)
{
	while(ino!=0)
	{
		printf("$ ");
		printf("* ");
		ino--;
	}
}

//Entry point function
int main()
{
	int i=0;
	printf("Enter your number : ");
	scanf("%d",&i);
	Print(i);
}

File: Java programming/problems on bitwise operator/problem13.java
/*

	13. Write a program which accept one number and position from user and on that bit. Return modified number. 
	
	  Input :  10 4
	  Output :  14 

	Author : Milind Nalawade
	
	Date : 03/08/2021
	
	Language : Java 4.0.2
	
*/
import java.lang.*;

class Logic 
{
	public static int ChkBit(int no,int pos)
	{
		int flag=0;
		flag=no|pos;
		return flag;		
	}
}

class problem13 extends Logic
{
	public static void main(String args[])
	{
		int no=Integer.parseInt(args[0]);
		int pos=Integer.parseInt(args[1]);
		int ret= ChkBit(no,pos);
		System.out.print("\n Output : "+ret+"\n");
	}
}